src/core/audio/AudioSoundIo.cpp

```diff
@@ -328,7 +326,6 @@ void AudioSoundIo::writeCallback(int frameCountMin, int frameCountMax)
 
 			for (int channel = 0; channel < layout->channel_count; channel += 1)
 			{
-				float sample = gain * m_outBuf[m_outBufFrameIndex][channel];
```

Comment:
Getting rid of `sample` entirely doesn't look right, as it's referenced in the next line.

(Interestingly, CI still passes, as we don't build this file for any platform. It looks like we try to build it on Linux, but we have a missing `libsoundio` dependency.)